cpp/src/io/parquet/parquet_gpu.hpp

```diff
@@ -375,6 +389,13 @@ struct ColumnChunkDesc {
   int32_t src_col_schema{};  // my schema index in the file
 };
 
+/**
+ * @brief A utility structure for use in decoding page headers.
+ */
+struct chunk_page_info {
+  PageInfo* pages;
+};
```

Comment:
Is this redundant? So the struct contains a pointer, and we will use pointer of pointer?

Reply:
See https://github.com/rapidsai/cudf/pull/14360#discussion_r1459833187